Archisinal.md

```diff
@@ -0,0 +1,310 @@
+**Artem Smolyarenko** - UI/UX designer with over 3 years experience. He specializes in brand development in all aspects and have an excellent understanding of typography, color psychology, as well as grid, web and mobile principles. Artem worked on different types of products with startups and corporate teams from around the world. Skills: UI/UX design, product design, web and mobile app design. Tech stack & tools: Figma, Adobe Photoshop, Adobe Illustrator, Adobe XD.
+
+### Team Code Repos
+
+- **Archisinal** [https://github.com/Archisinal](https://github.com/Archisinal)
+- **Danylo Semirazov** - [github.com/sventime](https://github.com/sventime), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+- **Oleh Kulykovskyi**  - [github.com/blindbeat](http://github.com/blindbeat), [gitlab.com/asylum-space](https://gitlab.com/asylum-space)
+
+### Team LinkedIn Profiles (if available)
+
+- **Tom Humber** https://www.linkedin.com/in/tom-humber-b65888118
+- **Danylo Semirazov** - [linkedin.com/in/danylosemirazov/](https://www.linkedin.com/in/danylosemirazov/)
+- **Oleh KulyKovskyi**  - [linkedin.com/in/oleh-kulykovskyi-329523234/](http://www.linkedin.com/in/oleh-kulykovskyi-329523234/)
+
+
+## Development Status :open_book:
+
+- [Medium](https://medium.com/@archisinal)
+- [Twitter](https://twitter.com/archisinal)
+- [Instagram](https://www.instagram.com/archisinal.io/)
+
+## Development Roadmap :nut_and_bolt: ##
+
+### Overview
+
+The development roadmap for Archisinal includes multiple milestones to ensure the successful launch and operation of the platform.
+
+### **Overview**
+
+- **Total Estimated Duration:**
+    - MVP: 2.5 months
+    - Post-MVP: 9 months
+    - Post-Launch: 10 months
+- **Full-Time Equivalent (FTE): 6**
+- **Total Costs:**
+    - MVP: $60,000
+
+## MVP (brief discussion)
+
+### **Milestone 1: Smart Contract Development**
+
+- **Estimated duration:** 1 month
+- **FTE:** 3
+- **Costs:** 24,000 USD
+
+**Overview:**
+
+- Develop smart contracts for NFT minting, buying, selling, and royalty distribution
+- Testing and deployment on Astar testnet Shibuya
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how users can interact with the developed smart contracts. |
+| 0c. | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit & integration tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains the development and features of the smart contracts, as well as any challenges faced and lessons learned. |
+| 1. | NFT Minting | Develop a smart contract functionality for NFT minting when creators upload their designs. |
```

Comment:
Where are you going to store all the metadata associated with the NFTs?

Reply:
We plan to implement the PSP24Metadata trait, which offers a simple key-value struct for defining metadata. Considering the specifics of the project, this metadata is not expected to change frequently. Therefore, we will primarily store the IPFS link to the project's metadata on-chain. The metadata will include the project's name, description, media file links, project source links, etc. The creator will be able to update this IPFS link after confirming with the new project owner.

Reply:
Do you mean [PSP34Metadata](https://github.com/w3f/PSPs/blob/master/PSPs/psp-34.md#psp34metadata)?

Reply:
Sure, typo). It’s PSP34Metadata.